src/index.ts

```diff
@@ -64,7 +57,7 @@ const VERSION = {
 type ObjectValues<T> = T[keyof T]
 type Version = ObjectValues<typeof VERSION>
 
-class Storyblok {
+class Storyblok<Content = ISbContentType> {
```

Comment:
Note that providing a default value like `= ISbContentType` makes this a non-breaking change, because that was the default used previously.